test/newsimplification/Simplifier.spec.ts

```diff
@@ -0,0 +1,147 @@
+import {simplify, Simplifier} from '../../src/newsimplification/Simplifier';
+import {isModelObject, simplifyName} from '../../src/newsimplification/Utils';
+import simplifyProperties from '../../src/newsimplification/SimplifyProperties';
+import simplifyAllOf from '../../src/newsimplification/SimplifyAllOf';
+import { CommonModel, Schema } from '../../src/models';
+
+let mockedIsModelObjectReturn = false;
+jest.mock('../../src/newsimplification/Utils', () => {
```

Comment:
Jest is commonly used in frontend but it is heavy, consider using mocha for backend tests.

Reply:
Good point, me and @magicmatatjahu have also discussed it, the mocking framework of jest is also not the greatest IMO..

Reply:
But it works for now, so if you have a strong opinion feel free to create an issue 😄